### DISCUSSION TITLE
Do not connect AR.js

### DISCUSSION THREAD

**Discussion by mianger22** [AR-js-org/AR.js#618] [Q&A]
Good afternoon. How to connect AR.js, if the official link is _https://raw.githack.com/AR-js-org/AR .js/master/aframe/build/aframe-ar.js_ not working?

**Comment by nickw1** [comment]
Hi, you have an incorrect space in the URL.

The correct URL is https://raw.githack.com/AR-js-org/AR.js/master/aframe/build/aframe-ar.js.

**Reply by mianger22** [reply]
No, it's not about the url. I made a gap in this question, but I use the correct address in the project, and it still doesn't work. Maybe there are similar services?

**Comment by nickw1** [comment]
@mianger22 so what happens? Do you mean you cannot request that URL?
I tried it just now from my machine, it works fine (i.e. I get a JavaScript bundle back).

Are any errors reported in your developer console?

Thanks.

**Reply by kalwalt** [reply]
I can confirm that it works also to me.

**Reply by mianger22** [reply]
In most cases, when I click on the link, I get the message "Unable to connect to the site." And in rare cases, everything is fine, the code can be downloaded. 

Please see how to solve this problem. Otherwise, AR cannot be connected via the link, and because of this, no projects work.

**Reply by nickw1** [reply]
I'm not sure what's happening here - have you tried clearing your cache?

**Reply by mianger22** [reply]
![image](https://github.com/user-attachments/assets/dd60aa64-afd7-48f9-88b0-a7e8da1f8490)

Cleared the cache

**Reply by nickw1** [reply]
(For info: Translates as "unable to connect to site").

I have no problem, is your ISP or your work IT department (if this is a work machine) blocking access to certain sites?
